Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchSubmission.java

```diff
@@ -74,10 +74,11 @@ public String getInstallationID() {
     }
 
     @NotNull
-    protected CloseableHttpClient getHttpClient() throws IOException {
+    public CloseableHttpClient getHttpClient() throws IOException {
         return HttpClients.custom()
-                .useSystemProperties()
-                .setDefaultCredentialsProvider(credentialsProvider).build();
+                 .useSystemProperties()
```

Comment:
only format the "build" line

---
In reply to: [234865336](https://github.com/Microsoft/azure-tools-for-java/pull/2348#discussion_r234865336) [](ancestors = 234865336)

Reply:
change protected to public for the method.since jobutils needs calling the method

---
In reply to: [234865336](https://github.com/Microsoft/azure-tools-for-java/pull/2348#discussion_r234865336) [](ancestors = 234865336)